docs/guides/serverless-hamsim-template.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<h1>Build a function template for Hamiltonian simulation</h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>